src/guess.js

```diff
@@ -0,0 +1,50 @@
+import compareNumbers from './compare-numbers.js';
+import changeTC from './changeTC.js';
+
+
```

Comment:
make sure you delete any unnecessary white space like this, just one line is fine